assets/src/dashboard/components/layout/scrollable.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import useLayoutContext from './useLayoutContext';
+import { SQUISH_LENGTH } from './provider';
+
+const ScrollContent = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
Maybe add a comment as to why `-webkit-overflow-scrolling: touch;` is being used?  My googling says it's to support smooth scrolling in iOS safari, but Chrome doesn't recognize it and I wouldn't want a future dev to delete it thinking it's not useful:
![image](https://user-images.githubusercontent.com/40646372/80517539-187a5400-893a-11ea-860b-4ef53ba9d7c8.png)